Bio/PDB/ResidueDepth.py

```diff
@@ -50,21 +50,18 @@
 
 """
 
-
+import numpy
 import os
+import subprocess
```

Comment:
Nice to sort these :)

Reply:
This should now be fixed in [this commit](https://github.com/amorehead/biopython/commit/b22226e5225866cc8a87b18d21da5cea5987e3d8).